query/user.js

```diff
@@ -0,0 +1,19 @@
+const { sql } = require("slonik");
+
+const getUserData = async (db, {username}) => {
+	try {
+		const { rows: user } = await db.query(sql`
```

Comment:
Mejor usad `db.maybeOne` o `db.one` en lugar de `db.query`. Si solo queréis un usuario, mejor que os devuelva un objeto directamente en lugar de un array de un solo elemento